apps/extension/src/routes/page/onboarding/generate.tsx

```diff
@@ -60,6 +114,21 @@ export const GenerateSeedPhrase = () => {
               isSuccessCopyText
             />
           </div>
+
+          {reveal && (
+            <div className='mt-4 rounded-lg border border-gray-500 bg-gray-800 p-4 shadow-sm'>
+              <h4 className='text-lg font-semibold text-gray-200'>Important!</h4>
+              <p className='mt-2 text-gray-300'>
+                Block Height:{' '}
+                <span className='font-bold text-gray-100'>{blockHeight ?? 'Loading...'}</span>
+              </p>
+              <p className='mt-2 text-sm text-gray-400'>
+                Please save the wallet creation height along with your recovery passphrase. It will
+                help you restore your wallet more quickly!
```

Comment:
suggestion:

````diff
- It will help you restore your wallet more quickly!
+ It's not required, but will help you restore your wallet quicker on a fresh Prax install next time.
```

Reply:
reworded comment